pkg/controller/plan/adapter/ocp/validator.go

```diff
@@ -179,3 +179,8 @@ func (r *Validator) NetworksMapped(vmRef ref.Ref) (ok bool, err error) {
 
 	return true, nil
 }
+
+// NO-OP
+func (r *Validator) DirectStorage(vmRef ref.Ref) (bool, error) {
+	return true, nil
```

Comment:
it's actually interesting to see how CDI handles direct LUNs, maybe it shouldn't remain NO-OP for OCP-to-OCP
@bennyz

Reply:
ok, we wrote below "current, oVirt-only" so let's think about it separately